.prettierignore

```diff
@@ -1,3 +1,4 @@
 CLA.md
 CODE_OF_CONDUCT.md
 LICENSE.md
+PULL_REQUEST_TEMPLATE.md
```

Comment:
Why?

Reply:
Prettier had some warnings, so I ran prettier fix and it reverted the change

Reply:
@FARKIr I think you are lost, as your question is not related to anything in this pull request. If you are unsure where to go, please check out our help page here: <https://www.home-asistant.io/help>.

Thanks 👍
 
../Frenck